packages/builder/package.json

```diff
@@ -29,16 +29,32 @@
     "test": "node ../../node_modules/mocha/bin/mocha",
     "coverage": "node ../../node_modules/nyc/bin/nyc.js ../../node_modules/mocha/bin/mocha",
     "build": "node ../../node_modules/typescript/bin/tsc --build .",
-    "clean": "node ../../node_modules/rimraf/bin.js builtin-tasks internal *.d.ts *.map *.js tsconfig.tsbuildinfo build"
+    "clean": "node ../../node_modules/rimraf/bin.js builtin-tasks internal *.d.ts *.map *.js tsconfig.tsbuildinfo build",
```

Comment:
why using a `rimraf`? Can we use `rm`?

Reply:
It was used by the original project. I had no intention to clean up all of the parts of the project.